packages/jsii-calc/lib/compliance.ts

```diff
@@ -1682,3 +1681,35 @@ export abstract class VoidCallback {
     }
     protected abstract overrideMe(): void;
 }
+
```

Comment:
what about @stability? It also seems like there's also a test that has @stable in there... Are those two different things?